tests/test_playbooks/setting.yml

```diff
@@ -33,24 +39,28 @@
   - include: tasks/setting.yml
     vars:
       setting_name: excluded_facts
-      setting_value: foo,bar
-      setting_value_override: ['bar', 'foo']
+      setting_value: ['foo', 'bar']
       expected_change: true
   - include: tasks/setting.yml
     vars:
       setting_name: excluded_facts
-      setting_value: foo,bar
-      setting_value_override: ['bar', 'foo']
+      setting_value: ['foo', 'bar']
       expected_change: false
 
   - include: tasks/setting.yml
     vars:
-      setting_name: content_action_accept_timeout
+      setting_name: token_duration
       setting_value: 99
+      setting_value_override: "99"
```

Comment:
This is needed, because the 99 already arrives as a string in entity_dict. Maybe it is the jinja magic in tasks/setting.yaml, that converts it. ;(